client/webserver/site/src/js/forms.ts

```diff
@@ -335,7 +335,7 @@ export class NewWalletForm {
       })
     } else this.subform.update(configOpts, {})
 
-    if (this.subform.dynamicOpts.children.length) Doc.show(page.walletSettingsHeader)
+    if (this.subform.dynamicOpts.attributes.length) Doc.show(page.walletSettingsHeader)
```

Comment:
Children might be 0, as they will be hidden.

Reply:
`children` doesn't consider visibility or display, and `attributes` does not count descendants. Wanna check this again?

Reply:
You are correct. The Reason is because the children is at defaultSettings, because gap limit have a default value.

So now I am also considerent defaultSettings children for showing the wallet settings header.